pyproject.toml

```diff
@@ -120,7 +121,7 @@ commands =
 src = ["src", "tests"]
 line-length = 88
 extend-ignore = [
-    "E501",  # Line too long
+    "E501", # Line too long
```

Comment:
I would also leave this the way it was for git blame purposes

Reply:
I think it's fine, we'll only end up with other developers doing the same thing later anyway

Reply:
I agrre with Dom